invenio_search_ui/static/templates/invenio_search_ui/marc21/default.html

```diff
@@ -1,6 +1,7 @@
 <ul>
   <li ng-repeat="record in vm.invenioSearchResults.hits.hits track by $index">
-    <h4><a target="_self" ng-href="{{ record.links.self }}">{{ record.metadata.title_statement.title }}</a></h4>
+    {# FIXME find a better way to set the record url #}
```

Comment:
I'm not quite sure that it will work as in jinja, angular templates using regular html comments ``<!-- [...] -->``  :D